apps/examples/log_dump/log_dump_main.c

```diff
@@ -42,37 +52,47 @@ int main(int argc, FAR char *argv[])
 int log_dump_main(int argc, char *argv[])
 #endif
 {
+	if (argc != 2) {
```

Comment:
How about `argc > 2`?
I think we can support only `log_dump` for entire node fetching. same as `log_dump 0`

Reply:
`log_dump 0` is getting all of nodes. It looks little bit strange.

Reply:
done, now will show error log log dump 0,
```
TASH>>log_dump 0
TASH>>
Usage: log_dump [N]
Fetches the last N nodes of log dump

  N    The number of nodes to be fetched from the last (newly saved), N > 0
       If N is not provided then we fetch all the saved nodes
```
with ```log_dump``` we fetch all the nodes, with ```log_dump N```, ```N > 0```, we fetch the last N nodes